service/frontend/workflow_handler.go

```diff
@@ -5144,11 +5145,85 @@ func getBatchOperationState(workflowState enumspb.WorkflowExecutionStatus) enums
 	return operationState
 }
 
+func (wh *WorkflowHandler) ListDeployments(
```

Comment:
just pasting this here for easiness when resolving merge conflicts - my PR (https://github.com/temporalio/temporal/pull/6847/files) currently implements some of these read API's. If one of us does end up getting some conflicts, lets keep the implementation in the PR

Reply:
resolving since this is handled by merging deployment apis into versioning-3 separately